### DISCUSSION TITLE
Sable Admin 文档：路由设置 - WxW

### DISCUSSION THREAD

**Discussion by giscus** [Whbbit1999/wxw-blog-comments#11] [General]
# Sable Admin 文档：路由设置 - WxW

路由设置文件在 /src/router/models/ 路由 meta 说明路由 meta 可以书写一些路由的属性，ts 配置文件在 /types/typings.d.ts 12345678910111213141516171819202122232425import "vue-router";declare module "vue-router" &#12

https://www.whbbit.cn/wiki/sable/%E8%B7%AF%E7%94%B1%E8%AE%BE%E7%BD%AE.html

**Comment by Whbbit1999** [comment]
路由部分考虑进行重构
